packages/sirius-web/frontend/sirius-web/src/widgets/ReferenceWidgetDocumentTransform.ts

```diff
@@ -0,0 +1,244 @@
+/*******************************************************************************
```

Comment:
Should be moved to `sirius-web-application`

Reply:
Done